frontend/src/components/common/datepicker/datepicker.tsx

```diff
@@ -41,6 +41,7 @@ const Datepicker: FC<Props> = ({ control, name, label, errors }) => {
         showMonthDropdown
         dropdownMode="select"
         dateFormat="dd.MM.yyyy"
+        placeholderText="Enter date of birth"
```

Comment:
Let's receive the placeholder from props, to make the component reusable

Reply:
agreed